packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -2451,6 +2574,25 @@ export class IntervalCollection<TInterval extends ISerializableInterval> extends
 		);
 	}
 
+	public findIntervalsWithStartInRange(startPosition: number, range: number): TInterval[] {
```

Comment:
we shouldn't implement these APIs directly on `IntervalCollection`. If someone wants to use them, they should create their own index and attach it.